input/fsh/profiles/RORLocation.fsh

```diff
@@ -2,43 +2,36 @@ Profile: RORLocation
 Parent: Location
 Id: ror-location
 Description: "Profil crée dans le cadre du ROR"
-* ^version = "2.4"
-* ^status = #active
-* ^date = "2022-06-30"
-* ^publisher = "ANS"
+
```

Comment:
Pourquoi est-ce qu'il n'y a pas de contrainte sur id et sur meta ici ?

Reply:
Pourquoi tu voudrais voir cette contrainte ? Est ce qu'on l'ajoute à tous les profils ?

Reply:
Car on la trouve dans RORHealthcareService. Savez-vous d'où vient cette contrainte @sdemeyANS ou @TristanKEREVAL ?

Ca serait bien que ce soit uniforme, j'ai pas d'arguments s'il faut garder ou enlever

Reply:
Non peut-être que @paulineheurtebise sait ?

Reply:
J'ai aucun souvenir d'une discussion sur ce sujet ou de prise de décision. J'ai rien dans mes notes et mes CR. 
Effectivement on doit uniformiser. Dans quel sens ? ajouter une contrainte sur Meta et id ca parait trop fort peut etre ?

Reply:
Aller on enlève et c'est vite rajouté au pire :)

Reply:
C'est noté, merci

Reply:
Contraintes sur Id et meta retiré de toutes les ressources